src/app/convai-checker.component.html

```diff
@@ -11,13 +11,14 @@
     [configurationInput]="demoSettings.configuration"
     [showPercentage]="demoSettings.showPercentage"
     [showMoreInfoLink]="demoSettings.showMoreInfoLink"
-    [score]="getMaxScore(analyzeCommentResponse)"
```

Comment:
It looks like this is still on the component, is that intentional?

Reply:
Yes, that's right. Re: the comment discussion yesterday, I'm removing score from data binding and updating via a function instead.

Reply:
Yup, that makes sense, I'm just confused on it still being an attribute for the component.

Reply:
Okay, I removed the @Input() annotation.